src/utils/ruleUrl.js

```diff
@@ -0,0 +1,15 @@
+const revision = "master";
```

Comment:
[note] If using a tag (e.g. `v4.2.0`), the implementation would be more complex. I think the HEAD version (`master`) is enough.

Reply:
Using `master` sounds good enough.